src/lib/web3-utils.js

```diff
@@ -0,0 +1,83 @@
+import { DEFAULT_CHAIN_NAME } from '../environment'
+
+import sha3 from 'js-sha3'
```

Comment:
:scissors: Super nitpicky, could we move this at the top of the file since it’s a global import?

Reply:
Yes, definitely! I tend to do that as well, but overlooked this one :)